composer.json

```diff
@@ -29,5 +29,10 @@
         "psr-4": {
             "EventSauce\\DoctrineMessageRepository\\Tests\\": "tests"
         }
+    },
+    "config": {
+        "platform": {
+            "php": "7.3.0"
```

Comment:
This is mostly here so that we don't accidentally do a `composer update` that locks dependencies to something we cannot use on 7.3.